libraries/standard/http/src/private/http_client_internal.h

```diff
@@ -0,0 +1,85 @@
+/* Standard Includes. */
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "http_client.h"
```

Comment:
This header shouldn't include the public-facing header file if there is no direct dependency in this file